client/js/store/modules/game.js

```diff
@@ -52,27 +53,56 @@ function handleGameResponse(context, jwres, resolve, reject) {
       return reject(jwres.body.message);
   }
 }
+
+/**
+ * @returns number of queens a given player has
+ * @param player is the player object
+ */
+function queenCount(player) {
+  return player?.faceCards?.reduce(
+    (queenCount, card) => queenCount + (card.rank === 12 ? 1 : 0),
+    0
+  );
+}
+
 const initialState = resetState();
 export default {
+  // TODO: Look in to namespacing -- currently this will not work due to a collision
+  // with the game socket also being named `game`. Potentially rename one or the other?
```

Comment:
These are probably the most important changes in here. With the move to break up GameView, it seems imperative that we move any shared getters and utilities out so they can be easily reused. Typically I would namespace here but the socket logic uses the same inferred `game` name that this vuex module would use.

It's probably worth talking through some options, but for now I just opted to not namespace it and use them as-is.

Reply:
What kind of namespacing do you envision? Are you talking about pulling these shared utilities into the vuex store and namespacing it's modules? Or putting them elsewhere and implementing some kind of custom namespacing solution? Something else?

Reply:
> Are you talking about pulling these shared utilities into the vuex store and namespacing it's modules?

That's exactly it. I'd like to see all shared logic that uses the game object or any of the store values hoisted back up to the store level.

Anything that isn't exactly store related, but calculates or otherwise does some logic we should pull those out to shared utilities and test them with Jest.

As for the namespacing, I'd like to use the name `game` for the game module, but we still would need to figure out what to rename the server side stuff which currently use game also. I may be missing some of the puzzle but those are the general points.

Reply:
Looking forward to a deeper dive into this.

I'm inclined to tackle module reorganization as a part of the switch to pinia, so we don't rearchitect the vuex store then throw it out altogether. I think I see the gist of what you mean but I don't actually follow how module namespacing relates to server side code. 

I think the switch to pinia would actually subvert the need for namespacing, because each module would be individually imported as its own store and locally assigned to a variable in the component(s) consuming it, which I think would prevent collisions. Does that address your concerns?